src/deluge/model/clip/clip.h

```diff
@@ -37,8 +37,10 @@ class StorageManager;
 class Serializer;
 class Deserializer;
 
-enum class ClipGroupType { NONE, EXCLUSIVE, SHARED };
-
+// Only shared is implemented, plan is exclusive would act like the official behaviour and select one clip, sequential
+// would play through them one after another
```

Comment:
Could this comment describe SHARED, since it's the one implemented? (Or refer elsewhere for the description?)